source/quickinstallationguide/qig.rst

```diff
@@ -453,7 +506,7 @@ the system VMs images.
   
    /usr/share/cloudstack-common/scripts/storage/secondary/cloud-install-sys-tmplt \
    -m /export/secondary \
-   -u http://download.cloudstack.org/systemvm/4.6/systemvm64template-4.6.0-kvm.qcow2.bz2 \
+   -u http://download.cloudstack.org/systemvm/4.11/systemvmtemplate-4.11.0-kvm.qcow2.bz2 \
```

Comment:
use the `4.11.1` template.

Reply:
I'm not sure where this has been pulled from - the QIG-Update-CentOS7 branch in my repo already has been updated to use the 4.11 templates.